src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -335,51 +351,257 @@ private synchronized void writeToConsole(Person person, int index, long time, bo
   }
 
   private long setDemographics(Person person, Demographics city) {
-    person.attributes.put(Person.CITY, city.city);
-    person.attributes.put(Person.STATE, city.state);
-    
-    String race = city.pickRace(person.random);
-    person.attributes.put(Person.RACE, race);
-    String ethnicity = city.ethnicityFromRace((String)person.attributes.get(Person.RACE), person);
-    person.attributes.put(Person.ETHNICITY, ethnicity);
-    String language = city.languageFromEthnicity((String) person.attributes.get(Person.ETHNICITY),
-        person);
-    person.attributes.put(Person.FIRST_LANGUAGE, language);
-
-    String gender = city.pickGender(person.random);
-    if (gender.equalsIgnoreCase("male") || gender.equalsIgnoreCase("M")) {
-      gender = "M";
-    } else {
-      gender = "F";
-    }
-    person.attributes.put(Person.GENDER, gender);
+	  if(Config.get("generate.households.mode").equals("true")) {
+		  System.out.println("Running in households mode");
+
+		  // Create map and read in the sampled SPEW csv file for Massachusetts 
+
+		  @SuppressWarnings("rawtypes")
+		  ArrayList<List> spewList = Demographics.getSpewList();
```

Comment:
I realize there's probably no way you could have known to use Checkstyle, but from the diff here it's easy to spot that this isn't consistent with the code style in the rest of the code. I'll add this info to the contributors guide as well, but please set up Checkstyle using the config file located at ./config/checkstyle/checkstyle.xml . From there you'll see warnings in your IDE and can set up a formatter that will automatically format the code in the right way. I can help you get all that setup.